MANIFEST.in

```diff
@@ -1,8 +1,11 @@
-include requirements/*.txt
-recursive-include openassessment/templates *.html
-recursive-include openassessment/xblock/static *
-recursive-include openassessment/locale *.po
-recursive-include openassessment/locale *.mo
 include LICENSE
 include AUTHORS
 include README.rst
+include openassessment/xblock/static/css/*.css
+include openassessment/xblock/static/js/openassessment*.min.js
+recursive-include openassessment/xblock/static/js/src *.js
+recursive-include openassessment/templates *.html
+recursive-include openassessment/locale *.po
+recursive-include openassessment/locale *.mo
+global-exclude */test*
+global-exclude */test?/*
```

Comment:
You may be wondering "Why both of these final two lines?" The answer is that `/` gets treated specially - the first pattern will match *most* of what we want, but numerous instances of `blahblah/test/__init__.py` were still being included. Using both lines gets us where we want.